@xen-orchestra/web-core/lib/components/card/CardTitle.vue

```diff
@@ -0,0 +1,52 @@
+<!-- v1.0 -->
+<template>
+  <div class="card-title">
+    <div class="main-content">
+      <div class="title typo h6-medium"><slot></slot></div>
```

Comment:
Just to be consistent with the code style and to avoid errors with the upcoming ESLint rules:
```suggestion
      <div class="title typo h6-medium"><slot /></div>
```